tests/ieee802154_hal/main.c

```diff
@@ -115,7 +115,7 @@ void _rx_done_handler(event_t *event)
     int size = ieee802154_radio_read(ieee802154_hal_test_get_dev(RADIO_DEFAULT_ID), buffer, 127, &info);
     if (size > 0) {
         /* Print packet while we wait for the state transition */
-        _print_packet(size, info.lqi, info.rssi);
+        _print_packet(size, info.lqi, info.rssi + IEEE802154_RADIO_RSSI_OFFSET);
```

Comment:
If this is intended to print the RSSI, it shouldn't include the offset

Reply:
Its not the rssi without the offset though

Reply:
Ahh sorry you are right